main/read.c

```diff
@@ -877,7 +877,7 @@ static vString *iFileGetLine (void)
 			parseLineDirective (vStringValue (File.line) + 1);
 		matchLanguageRegex (getInputLanguage (), File.line);
 
-		if (use_multiline)
+		if (use_multiline && File.allLines)
```

Comment:
is this a real issue you're fixing, or just a theoretical one?  I don't see how this is useful as on https://github.com/universal-ctags/ctags/pull/1631/files#diff-0a4427393487ed8d5f467beb7f72034fR863 `File.allLines` is created unconditionally, and AFAIK the allocation is checked and the program aborts if it fails, so `File.allLines` *has been* created.